Buildings/Controls/OBC/FDE/PackagedRTUs/BAScontroller.mo

```diff
@@ -0,0 +1,309 @@
+  annotation (Dialog(tab="Supply fan", group="Down duct pressure limits"));
+
+  parameter Real minDDSPset(
+  final unit="Pa",
+  final displayUnit="Pa",
+  final quantity="PressureDifference")=125
+  "Minimum down duct static pressure reset value"
+  annotation (Dialog(tab="Supply fan", group="Down duct pressure limits"));
+
+  // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-140,60},{-100,100}}),
+        iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput occReq
+    "Terminal unit occupancy requests"
+    annotation (Placement(transformation(extent={{-140,34},{-100,74}}),
+        iconTransformation(extent={{-140,34},{-100,74}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbcReq
+    "Terminal unit setback cooling requests"
+    annotation (Placement(transformation(extent={{-140,8},{-100,48}}),
+        iconTransformation(extent={{-140,8},{-100,48}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbhReq
+    "Terminal unit setback heating requests"
+    annotation (Placement(
+        transformation(extent={{-140,-20},{-100,20}}), iconTransformation(
+          extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput totCoolReqs
+    "Total terminal unit cooling requests"
+    annotation (Placement(
+        transformation(extent={{-140,-48},{-100,-8}}), iconTransformation(
+          extent={{-140,-48},{-100,-8}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput highSpaceT
+    "Highest space temperature reported from all terminal units"
+    annotation (Placement(transformation(extent={{-140,-74},{-100,-34}}),
+        iconTransformation(extent={{-140,-74},{-100,-34}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam
+    "Most open damper position of all terminal units" annotation (
+      Placement(transformation(extent={{-140,-100},{-100,-60}}),
+        iconTransformation(extent={{-140,-100},{-100,-60}})));
+
+  // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yOcc
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{100,32},{140,72}}),
+        iconTransformation(extent={{100,32},{140,72}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBC
+    "True when setback cooling mode is active"
+    annotation (Placement(transformation(extent={{100,6},{140,46}}),
+        iconTransformation(extent={{100,6},{140,46}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBH
+    "True when setback heating mode is active"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}}),
+        iconTransformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yMinOAflowSet
+    "active outside air flow set point sent to factory controller"
```

Comment:
`Active ...`

Reply:
Would be great to have an editor to review this, before sending to the team for review... :)

Reply:
We're our own editors, so it becomes a second nature to comment on these types of errors.